config/logging.php

```diff
@@ -65,6 +65,20 @@
             'replace_placeholders' => true,
         ],
 
+        'requests' => [
+            'driver' => 'single',
+            'path' => storage_path('logs/requests.log'),
+            'level' => env('LOG_LEVEL', 'debug'),
+            'replace_placeholders' => true,
+        ],
+
+        '404' => [
+            'driver' => 'single',
+            'path' => storage_path('logs/laravel.log'),
+            'level' => env('LOG_LEVEL', 'notice'),
+            'replace_placeholders' => true,
+        ],
```

Comment:
Because these logs just go into the standard file, you don't need to make a custom channel for this